src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Confirm.tsx

```diff
@@ -219,311 +176,37 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
       })
       setDaysToBreakEven(daysToBreakEven)
     })()
-  }, [accountId, asset, protocolFeeCryptoBaseUnit, opportunity?.apy, state?.deposit.cryptoAmount])
-
-  const getEstimateFeesArgs: () => Promise<EstimateFeesInput | undefined> =
-    useCallback(async () => {
-      if (isTokenDeposit) return
-      if (!accountId) throw new Error('accountId required')
-      if (isUtxoChainId(chainId) && !maybeFromUTXOAccountAddress) {
-        throw new Error('UTXO from address required')
-      }
-
-      if (!state?.deposit.cryptoAmount) {
-        throw new Error('Cannot send 0-value THORCHain savers Tx')
-      }
-
-      const amountCryptoBaseUnit = bnOrZero(state.deposit.cryptoAmount).times(
-        bn(10).pow(asset.precision),
-      )
-      const maybeQuote = await getMaybeThorchainSaversDepositQuote({ asset, amountCryptoBaseUnit })
-      if (maybeQuote.isErr()) throw new Error(maybeQuote.unwrapErr())
-      const quote = maybeQuote.unwrap()
-
-      const amountCryptoThorBaseUnit = toThorBaseUnit({
-        valueCryptoBaseUnit: amountCryptoBaseUnit,
-        asset,
-      })
-
-      setProtocolFeeCryptoBaseUnit(
-        toBaseUnit(
-          fromThorBaseUnit(amountCryptoThorBaseUnit.minus(quote.expected_amount_deposit)),
-          asset.precision,
-        ),
-      )
-
-      const memoUtf8 = quote.memo
-      return {
-        amountCryptoPrecision: state.deposit.cryptoAmount,
-        assetId,
-        from: maybeFromUTXOAccountAddress,
-        to: quote.inbound_address,
-        memo: supportedEvmChainIds.includes(chainId as KnownChainIds) ? toHex(memoUtf8) : memoUtf8,
-        sendMax: Boolean(!isUtxoChainId(chainId) && state?.deposit.sendMax),
```

Comment:
we don't currently have send max logic plumbed through the new hook, so this behavior changes

Reply:
While we don't explicitly handle max sends (as in, passing `sendMax` to chain-adapters so the UTXO pick algorithm used is the slightly different max sends one) we do have fees deduction at `<Deposit />` step on percent option click, meaning the previous logic was quite off... we would use an amount to display to the user, but have another amount in the end since `sendMax` alway takes precedence in `utxoSelect`.

Confirmed things are still working for max UTXO sends:

<img width="558" alt="image" src="https://github.com/shapeshift/web/assets/17035424/986501df-4170-4b49-a47d-2b802c11cc08">
<img width="535" alt="image" src="https://github.com/shapeshift/web/assets/17035424/876542e1-73b7-43d7-a0a6-8659b893da33">
<img width="550" alt="image" src="https://github.com/shapeshift/web/assets/17035424/94a2dbe0-d3ca-407d-9440-d9f88ceccea8">
<img width="1633" alt="image" src="https://github.com/shapeshift/web/assets/17035424/2f446f24-6cc4-4142-b786-68bd3325aa2c">

Technically not a full balance send, as can be seen from the change output here:

<img width="1147" alt="image" src="https://github.com/shapeshift/web/assets/17035424/56f861c8-5b8e-45fe-84aa-4baee83b8d91">